### PR TITLE
Added cloud volumes to requested resource calculation.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Added a test for multiple cloud volumes.

https://bugzilla.redhat.com/show_bug.cgi?id=1455349

@miq-bot add_label bug, fine/yes, euwe/yes
@miq-bot assign @gmcculloug

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@tinaafitz 
Please review

File: content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb [hunk 1]
```diff
@@ -231,6 +231,12 @@ def cloud_storage(args_hash)
             else
               flavor.root_disk_size.to_i + flavor.ephemeral_disk_size.to_i + flavor.swap_disk_size.to_i
             end
+  if args_hash[:resource].options[:volumes]
+    args_hash[:resource].options[:volumes].each do |v|
+      $evm.log(:info, "Adding volume to storage:  #{v.inspect} ")
+      storage += v[:size].to_i.gigabytes
```
Comment (mkanoor) [inline content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb]: @billfitzgerald0120 Is the size normalized for all providers?

Comment (billfitzgerald0120) [inline content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb]: @mkanoor Cloud volumes only applies to Openstack

**Comment by billfitzgerald0120** [comment]
@mkanoor Made changes as requested.

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit 051fcf1e01f3eb3a8631d76f12de1a15cc1f523d
Author: Madhu Kanoor <mkanoor@redhat.com>
Date:   Mon Aug 14 14:29:03 2017 -0400

    Merge pull request #138 from billfitzgerald0120/quota_cloud_volumes
    
    Added cloud volumes to requested resource calculation.
    (cherry picked from commit 70a6dedc2c10062d7817d2867fa9635fe6c43f34)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1481446
```

**Comment by simaishi** [comment]
Euwe backport (to manageiq repo) details:

```
$ git log -1
commit 34e190196e43b5085ec88a44db9a74d7ac434198
Author: Madhu Kanoor <mkanoor@redhat.com>
Date:   Mon Aug 14 14:29:03 2017 -0400

    Merge pull request #138 from billfitzgerald0120/quota_cloud_volumes
    
    Added cloud volumes to requested resource calculation.
    (cherry picked from commit 70a6dedc2c10062d7817d2867fa9635fe6c43f34)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1481870
```

**Pull request merged**